src/fileformats/file_format_registry.cpp

```diff
@@ -69,6 +69,31 @@ std::unique_ptr<FileFormat> FileFormatRegistry::unregisterFormat(const FileForma
 }
 
 
+std::unique_ptr<Importer> FileFormatRegistry::makeImporter(const QString& path, Map& map, MapView* view)
```

Comment:
`makeImporter()` signature differs from `FileFormat::makeImporter()` as well as neighboring `makeExporter()`. Suggest either explaining the difference or aligning interfaces.